source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -131,6 +131,11 @@ Changed default settings
 | :dovecot_plugin:ref:`fts_dovecot_max_triplets`        | 0                      | 200             |                                                             |
 +-------------------------------------------------------+------------------------+-----------------+-------------------------------------------------------------+
 
+Doveadm mailbox commands
+------------------------
+``USER `` environment is no longer supported. All mail commands require providing ``-u``, ``-F`` or ``-A`` parameter.
```

Comment:
Would adding "variable" here also make it clearer? Some places in docs do use only "environment", but I think in this case it is definitely an environment variable.